cms/migrations/0017_setup_home_page.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.10 on 2020-06-09 10:12
+
+from django.db import migrations
+
+DEFAULT_HOMEPAGE_PROPS = dict(title="Home Page", subhead="This is the home page")
+DEFAULT_SITE_PROPS = dict(hostname="localhost", port=80)
+
+
+def create_home_page(apps, schema_editor):
+    """
+    Create the home page
+    """
+    Site = apps.get_model("wagtailcore", "Site")
+    Page = apps.get_model("wagtailcore", "Page")
+    HomePage = apps.get_model("cms", "HomePage")
+    ContentType = apps.get_model("contenttypes.ContentType")
+    site = Site.objects.filter(is_default_site=True).first()
+    valid_home_page = Page.objects.filter(
+        content_type=ContentType.objects.get_for_model(HomePage)
+    ).first()
+    root = Page.objects.get(depth=1)
+    if valid_home_page is None:
+        valid_home_page = HomePage(**DEFAULT_HOMEPAGE_PROPS)
+        root.add_child(instance=valid_home_page)
+        valid_home_page.refresh_from_db()
+    if site is None:
+        Site.objects.create(
+            is_default_site=True, root_page=valid_home_page, **DEFAULT_SITE_PROPS
+        )
+    elif site.root_page is None or site.root_page != valid_home_page:
+        site.root_page = valid_home_page
+        site.save()
+    wagtail_default_home_page = Page.objects.filter(
+        depth=2, content_type=ContentType.objects.get_for_model(Page)
+    ).first()
+    if wagtail_default_home_page is not None:
+        wagtail_default_home_page.delete()
```

Comment:
You need to move all the existing children of the default wagtail home page under the new home page otherwise this migration will erase all page content from the CMS.